components/server/ee/src/prebuilds/prebuild-manager.ts

```diff
@@ -222,17 +222,10 @@ export class PrebuildManager {
                 }
             }
 
-            let organizationId = (await this.teamDB.findTeamById(project.id))?.id;
-            if (!user.additionalData?.isMigratedToTeamOnlyAttribution) {
-                // If the user is not migrated to team-only attribution, we retrieve the organization from the attribution logic.
-                const attributionId = await this.userService.getWorkspaceUsageAttributionId(user, project.id);
-                organizationId = attributionId.kind === "team" ? attributionId.teamId : undefined;
-            }
-
             const workspace = await this.workspaceFactory.createForContext(
                 { span },
                 user,
-                organizationId,
+                project.teamId,
```

Comment:
The organizationId associated with the prebuild must be the same as the project's orgId.